wandb/sdk/wandb_run.py

```diff
@@ -1062,11 +1066,13 @@ def log_code(
             name: (str, optional) The name of our code artifact. By default, we'll name
                 the artifact `source-$PROJECT_ID-$ENTRYPOINT_RELPATH`. There may be scenarios where you want
                 many runs to share the same artifact. Specifying name allows you to achieve that.
-            include_fn: A callable that accepts a file path and
+            include_fn: A callable that accepts a file path and (optionally) root path and
```

Comment:
I found this docstring was stale; but took some liberties updating it... please review carefully :)